CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ database. Use a text editor like emacs or NotePad instead._
 
 **Inclusion criteria**
 
-- [ ] Make sure that any faculty you add meet the inclusion
+- [X] Make sure that any faculty you add meet the inclusion
```

Comment:
This whole file needs to not be present in the PR. DO NOT COMMIT A DELETION - please just remove it from the PR or open a new PR without this change.